service/src/main/resources/templates/ehr_narrative_statement_template.mustache

```diff
@@ -4,5 +4,18 @@
         <statusCode code="COMPLETE" />
         <availabilityTime value="{{availabilityTime}}" />
         <text>{{comment}}</text>
+        {{#hasReference}}
+        <reference typeCode="REFR">
+            <referredToExternalDocument classCode="DOC" moodCode="EVN">
+                <id root="{{narrativeStatementId}}"/>
+                <code code="25821000000100" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Original Text document">
+                    <originalText>Record Attachment</originalText>
+                </code>
+                <text mediaType="{{referenceContentType}}">
+                    <reference value="file://localhost/{{referenceTitle}}"/>
```

Comment:
Are we okay referencing local files?

Reply:
This was taken from the mapping spreadsheet, so I'm not sure? @mattd-kainos maybe you can help.
I've not added some of the logic around attachment handling, as I believe this will be done in another ticket, or if not then I can add this?

Reply:
the file://localhost/ is just a bit of boilerplate that needs to be included by convention

We need to generate filenames here according to the "Normal / Absent Attachment Handing" in the mapping spreadsheet. Everything about the manifest, or about creating the attachments themselves are out of scope for this story.

Since we have also not done a MIME Type -> file extension mapping yet you can use `.xyz` for every file extension.

Reply:
I've now set a default file extension